_examples/kafka-to-http/README.md

```diff
@@ -0,0 +1,32 @@
+# HTTP producer example
+
+This example showcases the use of HTTP producer to call webhooks with HTTP POST requests. It consists of three services:
+
+1. `producer` publishes messages on Kafka. The messages come in three varieties: `Foo`, `Bar`, and `Baz`. The event type is encoded in metadata, under the key `event_type`.
+1. `webhook_server` is a HTTP server that listens for requests and prints the path, method, and payload on stdout.
```

Comment:
it is inconsisten with the directorny name (`http-server`), it's good idea to unify it